src/components/manifold-plan-table/manifold-plan-table.tsx

```diff
@@ -216,23 +215,18 @@ export class ManifoldPlanTable {
   handleCtaClick(e: MouseEvent, planId: string, destination = '') {
     e.preventDefault();
 
-    analytics(
-      {
-        description: 'Track manifold-plan-table CTA clicks',
+    this.connection.analytics
```

Comment:
Since this is async, do we need to change the type to `connection?: Connection` above? Will `connection.analytics` throw an error if it hasn’t initialized yet?

Reply:
Well `componentWillLoad` currently waits for it so there's no way we can get here without this being initialized. We did talk about not waiting for it on load, but even in that case you wouldn't be fetching data until it's initialized, so there would be no button to click I think?